react-ui/src/pages/contest/ContestAppBarMain.jsx

```diff
@@ -6,11 +6,12 @@ import Box from '@material-ui/core/Box';
 import PropTypes from 'prop-types';
 import { useLocation } from 'react-router-dom';
 
-import { useVotingStatus } from '../../common';
+// eslint-disable-next-line import/no-unresolved
```

Comment:
Why is this needed?

Reply:
Removed this and another I found in `Home`